src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -29,7 +39,29 @@
 
 
 class RadioToolButton(Gtk.RadioToolButton):
-    """An implementation of a "push" button."""
+    '''
+    An implementation of a "push" button. GObject properies that can
+    be set are:
+        accelerator (string)
+        tooltip (string)
+        xocolor (:class:`XoColor`)
+        hide_tooltip_on_click (bool)
+
+    Args:
+        icon_name (string): name of icon to be used
+
+    Keyword Args:
+
+        accelerator (string): keyboard shortcut to be used to
+        activate this button
+
+        tooltip (string): tooltip to be displayed when user hovers over button
+
+        xo_color (:class:`XoColor`): XoColor of button
```

Comment:
Just a styntax thing, it only needs to be `(sugar3.graphics.xocolor.XoColor)`, no special `:class:`